docs/content/en/docs/reference/cloudstack/cloudstack-prereq.md

```diff
@@ -5,3 +5,63 @@ weight: 15
+* d2glxqk2uabbnd.cloudfront.net (http://d2glxqk2uabbnd.cloudfront.net/) (for EKS Anywhere and EKS Distro ECR container images)
+* api.ecr.us-west-2.amazonaws.com (http://api.ecr.us-west-2.amazonaws.com/) (for EKS Anywhere package authentication matching your region)
+* d5l0dvt14r5h8.cloudfront.net (http://d5l0dvt14r5h8.cloudfront.net/) (for EKS Anywhere package ECR container images)
+* api.github.com (http://api.github.com/) (only if GitOps is enabled)
+
+### CloudStack information needed before creating the cluster
+
+You need at least the following information before creating the cluster.
+See [CloudStack configuration]({{< relref "../clusterspec/cloudstack/" >}}) for a complete list of options and [Preparing CloudStack]({{< relref "./cloudstack-preparation/" >}}) for instructions on creating the assets.
+
+* *Static IP Addresses*: You will need one IP address for the management cluster control plane endpoint, and a separate one for the controlplane of each workload cluster you add.
+
+Let’s say you are going to have the management cluster and two workload clusters. For those, you would need three IP addresses, one for each. All of those addresses will be configured the same way in the configuration file you will generate for each cluster.
+
+A static IP address will be used for each control plane VM in your EKS Anywhere cluster. Choose IP addresses in your network range that do not conflict with other VMs and make sure they are excluded from your DHCP offering.
+An IP address will be the value of the property controlPlaneConfiguration.endpoint.host in the config file of the management cluster. A separate IP address must be assigned for each workload cluster.
+
+* CloudStack datacenter: You need the name of the CloudStack Datacenter plus the following for each Availability Zone (availabilityZones). Most items can be represented by name or ID:
+    * Account (account): Account with permission to create a cluster (optional, admin by default).
+    * Credentials (credentialsRef): Credentials provided in an ini file used to access the CloudStack API endpoint. See [CloudStack Getting started]({{< relref "../../getting-started/production-environment/" >}}) for details.
+    * Domain (domain):  The CloudStack domain in which to deploy the cluster (optional, ROOT by default)
+    * Management endpoint (managementApiEndpoint): Endpoint for a cloudstack client to make API calls to client. 
+    * Zone network (zone.network): Either name or ID of the network. 
+* CloudStack machine configuration: For each set of machines (for example, you could configure separate set of machines for control plane, worker, and etcd nodes), obtain the following information. This must be predefined in the cloudStack instance and identified by name or ID:
+    * Compute offering (computeOffering): Choose an existing compute offering (such as m4-large), reflecting the amount of resources to apply to each VM.
```

Comment:
Can we use something like `large-instance` as example and not m4-large? m4-large might not necessarily translate well for cloudstack users.

Reply:
Done.